notebooks/Accessing_File_with_GDS.ipynb

```diff
@@ -498,7 +459,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.10"
+   "version": "3.6.9"
```

Comment:
It might be worthwhile to use 3.7 or 3.8 instead. RAPIDS dropped 3.6 support a while ago

Reply:
The version number in metadata was automatically generated while using jupyter notebook to edit .ipynb file.
Updated to 3.8.8 manually.